target/linux/ipq40xx/config-5.4

```diff
@@ -527,7 +525,6 @@ CONFIG_UBIFS_FS_ZSTD=y
 CONFIG_UEVENT_HELPER_PATH=""
 CONFIG_UNCOMPRESS_INCLUDE="debug/uncompress.h"
 CONFIG_UNWINDER_ARM=y
-# CONFIG_UNWINDER_FRAME_POINTER is not set
```

Comment:
You're making a lot of changes to this config which don't seem to be directly related to this device.

Reply:
It's not me manually doing those, but make kernel_menuconfig will refresh the config as well based on generic config changes.
IPQ40xx has not been refreshed for a while so that's the reason.

Reply:
Maybe do a refresh in a separate commit beforehand. I just tried it on master, and I was prompted for CONFIG_CGROUPS ...

Reply:
I just pushed a refresh myself ...

Reply:
Thanks, I was just about to do it.